mm2src/coins/utxo.rs

```diff
@@ -22,7 +22,7 @@
 //
 
 pub mod bch;
-mod bchd_grpc;
+pub mod bchd_grpc;
```

Comment:
As I can see this module isn't used outside of `coins` crate. So we can downgrade it's visibility to `pub(crate)`